static/app/views/eventsV2/queryList.spec.jsx

```diff
@@ -11,6 +11,7 @@ import {DashboardWidgetSource, DisplayType} from 'sentry/views/dashboardsV2/type
 import QueryList from 'sentry/views/eventsV2/queryList';
 
 jest.mock('sentry/actionCreators/modal');
+jest.mock('sentry/components/charts/eventsRequest');
```

Comment:
One of the deeper components that used `eventsRequest` was erroring because a response was not a list (it was trying to access `.length`). Mocking since none of the tests seem to depend on it.